docs/json_schemas/fiveg_n2/v0/provider.json

```diff
@@ -0,0 +1,41 @@
+{
+  "title": "ProviderSchema",
+  "description": "Provider schema for fiveg_n2.",
+  "type": "object",
+  "properties": {
+    "unit": {
+      "$ref": "#/definitions/BaseModel"
+    },
+    "app": {
+      "$ref": "#/definitions/FivegN2ProviderAppData"
+    }
+  },
+  "required": [
+    "app"
+  ],
+  "definitions": {
+    "BaseModel": {
+      "title": "BaseModel",
+      "type": "object",
+      "properties": {}
+    },
+    "FivegN2ProviderAppData": {
+      "title": "FivegN2ProviderAppData",
+      "type": "object",
+      "properties": {
+        "amf_hostname": {
+          "title": "Amf Hostname",
```

Comment:
OAI expects an IP address not a hostname.

Reply:
Do we care to keep the hostname if the RAN impl of our choice doesn't need it?

Reply:
I guess provider should be setting both and the requirer can choose whichever it needs.